.travis.yml

```diff
@@ -17,6 +17,7 @@ matrix:
     - env: PGVERSION=10
     - env: PGVERSION=11
     - env: PGVERSION=12
+    - env: PGVERSION=13
     - env: LINTING=true
 before_install:
   - git clone -b update-uncrustify-on-travis --depth 1 https://github.com/citusdata/tools.git
```

Comment:
we usually use a tag instead of branch

Reply:
I think we should create a new tag on tools develop branch after https://github.com/citusdata/tools/pull/78 gets merged. Then use the new tag name in here.

Possibly `0.7.14` or `0.8.0` maybe? I don't really understand the versioning schema on that repository.